ccm_web/server/src/api/api.utils.ts

```diff
@@ -74,7 +91,7 @@ export function createLimitedPromises<T> (
   const limit = pLimit(maxConcurrentPromises)
   const limitedPromises = funcs.map(async (func) => {
     return await limit<[], T>(async () => {
-      logger.debug(`Promises still in queue: ${limit.pendingCount}`)
+      logger.debug(`Promises still in queue: ${String(limit.pendingCount)}`)
```

Comment:
Was there a linting error here, or did Codacy flag something? Codacy can't handle types from libraries correctly. I didn't see any issues in my IDE for this. The `String()` usage may not be hurting anything, but it may be redundant for simple types like `number`.

Reply:
Yes, this was in response to a lint problem.  I believe it was in VSC's linting.

After fixing that problem, **_more_** lint problems were raised for that function.  I wasn't sure how to resolve them, so I decided to let them sit while I responded to other things, like where/how the Cirrus' URL is defined.

Looking back at that code now, those lint problems are gone.  Furthermore, if I remove the `String` casting now, the earlier lint problem about it doesn't reappear for some reason.

Perhaps something else changed in the code.  Maybe something was there that confused eslint and caused it to give me those errors, but now the conditions are better and all is OK.

I'll remove the `String` casting.

Reply:
🏁